docs/source/using/addons.adoc

```diff
@@ -75,6 +77,22 @@ This can be useful in cases where you know that a command such as `oc` might tak
 
 NOTE: Trying to use an undefined command will cause an error when the add-on gets parsed.
 
+[[addon-openshift-version-semantics]]
+== OpenShift Version Semantics
+
+Within an add-on, one can enforce the OpenShift version which needs to be running in order to apply the add-on.
```

Comment:
"As part of the add-on metadata you can specify the OpenShift version which needs to be running in order to apply the add-on."